bin/run-lerna.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script to run `lerna` command with enviornment variable
+ * `LERNA_ROOT_PATH` set to the root directory of `loopback-next` monorepo
+ */
+'use strict';
+
+const path = require('path');
+const lerna = require('lerna');
+const build = require('../packages/build');
+
+/**
+ * Check if the script is running within Visual Studio Code
+ */
+function isVSCode() {
+  return Object.keys(process.env).some(k => k.indexOf('VSCODE') === 0);
+}
+
+function run(argv, options) {
+  const ls = new lerna.LsCommand(
+    null,
+    {json: true, loglevel: 'silent'},
+    path.join(__dirname, '..'),
+  );
+  const rootPath = ls.repository.rootPath;
+
+  process.env.LERNA_ROOT_PATH = rootPath;
+  let args = argv.slice(2);
+
+  // Force lerna command runs in non-parallel mode for VSCode builds
```

Comment:
I am not very happy with this solution, it will significantly slow down builds triggered from VS Code. Can we either leave this part out of this initial pull request?

Here is what I would like us to eventually implement:
 - At minimum, have two VSCode tasks - one for the usual "fast" build that reports broken error paths, anther for a "slow" build that works well with PROBLEMS window.
 - Ideally, we should invoke typescript in parallel via some sort of a tool that can merge the output in a clever way, or perhaps tell Lerna to not print those prefixes.

Either way, I'd like to have the option to run the build in parallel mode, it stays my default mode most of the time.

Reply:
I misunderstood what you want for VSCode. 

Here is what I found out:

- Lerna passes the package name as a tag to `strong-log-transformer` (from strongloop!). See https://github.com/lerna/lerna/blob/2.x/src/ChildProcessUtilities.js#L44-L45
- I can try to create a patch against lerna

I'll remove the VSCode check for now.